packages/fury/test/parse-test.js

```diff
@@ -17,64 +18,99 @@ describe('Parser', () => {
     fury.use(adapter);
   });
 
-  it('should parse through mediatype', (done) => {
-    fury.parse({ source: 'dummy', mediaType: 'text/vnd.passthrough' }, (err, result) => {
-      assert.equal(result.content, 'dummy');
-      done(err);
+  describe('using callback', () => {
```

Comment:
Diff noise, I've only indented all these tests to be inside the 'using callback' describe.